config/physical_location_codes.yml

```diff
@@ -9,3 +9,12 @@ publicpolicy: 'MUDD'
 univarchives: 'MUDD'
 mudd: 'MUDD'
 rcpph: 'MUDD'
+scarcpxm: 'Remote Storage (ReCAP): Manuscripts. Special Collections Use Only'
+scactsn: 'Cotsen Children’s Library Archival. Special Collections Use Only'
+scagax: 'Graphic Arts Archival. Special Collections Use Only'
+scamss: 'Manuscripts Archival. Special Collections Use Only'
+scawa: 'Western Americana Archival. Special Collections Use Only'
+scaex: 'Rare Books Archival. Special Collections Use Only'
+scahsvm: 'Manuscripts Archival. Special Collections Use Only'
+scamudd: 'Mudd Archival. Special Collections Use Only'
+scathx: 'Theater Archival. Special Collections Use Only'
```

Comment:
@regineheberlein Are these values correct? I wasn't sure if I should use Internal or External names from the Location Proposal Checklist document.

Reply:
Thanks @eliotjordan! The archivists are applying the internal name (i.e. code) to the description, and we'll want to expand that for display on the finding aids site and also make sure the code gets passed on to Aeon for requesting. 

I'm not entirely sure if I'm answering your question, let me know if not!

Reply:
According to this [Google Doc](https://docs.google.com/document/d/1E36NShlpqeq7zFXmMcP1Tu5ZD2NWgySgp20teTGiQWE/edit#), there is an Internal Name for staff:
 `scaex: 'Rare Books Archival'`

and an External Name for public display:
`scaex: 'Rare Books Archival. Special Collections Use Only'` 

I chose the external name. It sounds like we should use the internal name instead?